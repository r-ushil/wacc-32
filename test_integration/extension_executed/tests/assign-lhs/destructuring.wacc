begin
  pair(int, int) newpair(a, b) = newpair(1, 2);

  println a;
  println b
end
