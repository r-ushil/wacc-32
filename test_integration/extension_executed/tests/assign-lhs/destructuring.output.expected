Compiler exit code: 0
--------- stdout ---------
1
2
--------------------------
Exit code: 0
